### ISSUE TITLE
Add support for paginated pages (pagebrowsers) in header tags

### ISSUE THREAD

**Issue by thomaszbz** [mblaschke/TYPO3-metaseo#125]
by Georg Tiefenbrunn via https://forge.typo3.org/issues/62152

Hi Markus!

I found no way to alter (or disable) the creation of 

    <link rel="next" .../>, <link rel="prev" .../">
    and
    <link rel="canonical" .../">

tags.

On a page containing a plugin (e.g. EXT:news) which outputs paginated content I'd like the following output:

url: http://example.com/news/page/3/
canonical: http://example.com/news/page/3/
next: http://example.com/news/page/4/ (if there is a next page)
prev: http://example.com/news/page/2/
up: http://example.com/news/

atm canonical is http://example.com/news/ for all pages and next/prev are the pages from the pagetree.

Did I miss any option?

**Comment by thomaszbz** [comment]
.

**Comment by KreativeKrise** [comment]
As a workaround for the canonical you could disable the canonical functionality of metaseo with `plugin.metaseo.metaTags.useCanonical = 0` in your constants and  use plain TypoScript:

```
page.headerData {
	10 = TEXT
	10 {
		wrap = <link rel="canonical" href="|" />

		typolink {
			parameter.field = uid

			useCacheHash = 1
			
			addQueryString = 1
			addQueryString.method = GET
			addQueryString.exclude = L,id,cHash,no_cache

			returnLast = url
		}
	}
}
```

**Comment by thomaszbz** [comment]
I'm not sure if this is still an issue.

In the meantime, we disabled up/next/prev because at least google then thinks this is some sort of pagination or a fragmented text. In that case, google will lead visitors to the first page, as long as there is no version of the page containing all "fragments". That was done via #202.

@KreativeKrise What's the improvement of the canonical tag you provide via TypoScript, compared to the canonical tag metaseo 2.0.0 provides? Consider this to be a neutrally-posed question. Maybe we can add missing parts to metaseo.

At least metaseo's canonical tag can be switched on and off via TypoScript. And we have some fallback strategy for http/https to avoid duplicate content. However, there's still some issues left open in respect to the canonical tag.

**Comment by KreativeKrise** [comment]
@thomaszbz there is no improvement, it's just a workaround for the canonical problem @georg-tiefenbrunn had. But you are right, at least version 2.0.0 fixed the described behaviour!

**Comment by thomaszbz** [comment]
Fine, if it's fixed in metaseo 2.0.0 I'm going to close this issue.

If someone else has this problem after 2.0.0, please file a new issue and refer to #125.
